articles/quickstarts/get-started.md

```diff
@@ -27,13 +27,11 @@ To get up and running with the power of quantum, you can try out our tutorials:
 
 * [Quantum random number generator](xref:microsoft.quantum.quickstarts.qrng) - Start with a "Q# Hello World" style application, providing a brief introduction to quantum concepts while letting you build and run a quantum application in minutes.
 * [Explore entanglement with Q#](xref:microsoft.quantum.write-program) - This tutorial guides you on writing a Q# program that demonstrates some of the foundational concepts of quantum programming. If you are not ready to start coding, you can still follow along without installing the QDK and get an overview of the Q# programming language and the first concepts of quantum computing.
-* [Grover’s search algorithm](xref:microsoft.quantum.quickstarts.search) - Explore this example of a Q# program to get an idea of the power of Q# for expressing the quantum algorithm in a way that abstracts the low-level quantum operations.
-	This tutorial guides you through developing the program as a Q# application, using Visual Studio or Visual Studio Code.
+* [Grover’s search algorithm](xref:microsoft.quantum.quickstarts.search) - Explore this example of a Q# program to get an idea of the power of Q# for expressing the quantum algorithm in a way that abstracts the low-level quantum operations. This tutorial guides you through developing the program as a Q# application, using Visual Studio or Visual Studio Code.
 
-### Learning further
-* The [Microsoft Learn modules for quantum computing](https://docs.microsoft.com/learn/browse/?term=quantum) will teach you to master core concepts at your speed and on your schedule. You can learn the basics of how to create quantum programs with the QDK with our [first module](https://docs.microsoft.com/learn/modules/qsharp-create-first-quantum-development-kit/).
-* If you want to dive deeper into Q# programming, check out the [Quantum Katas](https://github.com/Microsoft/QuantumKatas) - a collection of self-paced programming exercises that introduce you to quantum computing via programming exercises in Q#.
-	Many of these katas are also available as Q# Notebooks. 
+## Learning further
+* Microsoft Learn offers free online training for quantum computing. The three-module Learning Path, [Quantum computing foundations](https://docs.microsoft.com/learn/paths/quantum-computing-fundamentals/), introduces the fundamental concepts of quantum computing and quantum algorithms, and gets you started building quantum programs using Q#.
```

Comment:
```suggestion
* Microsoft Learn offers free online training for quantum computing. The three-module Learning Path, [Quantum computing foundations](https://docs.microsoft.com/learn/paths/quantum-computing-fundamentals/), introduces the fundamental concepts of quantum computing and quantum algorithms, and gets you started building quantum programs using Q#.
```
Since we are going to add more modules maybe we should avoid mentioning the number of modules to avoid future inconsistencies.

Reply:
Good catch, thanks.